jobs/src/main/scala/dcos/metronome/model/CronSpec.scala

```diff
@@ -70,3 +77,44 @@ object CronSpec {
     }
   }
 }
+
+object CronSpecValidation {
+  val validDayOfMonth = "Day of the month must exist in the provided month and year (e.g. February have only < 29 days so running cron on 30.2. is invalid)"
+}
+
+/**
+  * In the existing validation logic inside cron-utils there is not validation that the given day exist in a month.
```

Comment:
this sentence is hard to read IMO.    Perhaps . "Cron-utils existing validation logic does NOT validate that a given day is possible for a given month."